src/Post.vue

```diff
@@ -55,21 +55,27 @@ export default {
       post: null,
       error: false,
       commentsEnabled: false
-    }
+    };
   },
-  created () {
+  created() {
     this.fetchBlogEntry();
   },
   watch: {
-    '$route': 'fetchBlogEntry'
+    $route: 'fetchBlogEntry'
   },
   methods: {
     fetchBlogEntry() {
       let vm = this;
       vm.error = vm.post = null;
       vm.loading = true;
-      let xhr= new XMLHttpRequest();
-      xhr.open('GET', 'https://api.github.com/repos/inexorgame/blog-data/contents' + this.$route.path + '.md', true);
+      let xhr = new XMLHttpRequest();
```

Comment:
shouldn't we use fetch/axios or similar to handle ajax requests? Just a thought to make development easier. But hey, whatever gets the job done.